dev/Pager/PrototypePager/PrototypePager.TestHooks.cs

```diff
@@ -14,5 +14,7 @@ public sealed partial class PrototypePager : Control
         public ComboBox ComboBoxDisplayTestHook;
         public ItemsRepeater NumberPanelDisplayTestHook;
         public Rectangle NumberPanelCurrentPageIdentifierTestHook;
+
+        public int AutoDisplayModeNumberOfPagesThresholdTestHook;
```

Comment:
>AutoDisplayModeNumberOfPagesThresholdTestHook [](start = 19, length = 45)

It looks like your test app doesn't use this expect to show the value.  In a production environment these are not free and I think we should avoid them if they aren't useful.  For the production version of this I would just hard code the value in the test app.

Reply:
I've removed it!